watertap/unit_models/zero_order/iron_and_manganese_removal_zo.py

```diff
@@ -0,0 +1,103 @@
+    """
+    Zero-Order model for an iron and manganese removal unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "iron_and_manganese_removal"
+
+        build_sido(self)
+
+        self.air_water_ratio = Var(self.flowsheet().time,
+                                units=pyunits.dimensionless,
+                                doc="ratio of air to water")
+
+        self.flow_basis = Var(self.flowsheet().time,
+                                   units=pyunits.m**3/pyunits.hour,
+                                   doc="flow basis")
+
+        self.air_flow_rate = Var(self.flowsheet().time,
+                                   units=pyunits.m ** 3 / pyunits.hour,
+                                   doc="air flow rate")
+
+        self.elec_coeff = Var(units=pyunits.hp/(pyunits.m ** 3 / pyunits.hour),
+                              doc="constant in electricity intensity equation")
+
+        self.filter_surf_area = Var(units=pyunits.m ** 2,
+                                    doc="dual media filter surface area")
+
+        self.num_filter_units = Var(units=pyunits.dimensionless,
+                                    doc="number of dual media filter units")
+
+        self.electricity = Var(self.flowsheet().config.time,
+                                units=pyunits.kW,
+                                bounds=(0, None),
+                                doc="Power consumption of iron and manganese removal")
+
+        self.electricity_intensity = Var(self.flowsheet().config.time,
+                                         units=pyunits.kWh / pyunits.m ** 3,
+                                         doc="Specific energy consumption with respect to feed flowrate")
+
+        self._fixed_perf_vars.append(self.air_water_ratio)
+        self._fixed_perf_vars.append(self.flow_basis)
+        self._fixed_perf_vars.append(self.elec_coeff)
+        self._fixed_perf_vars.append(self.filter_surf_area)
+        self._fixed_perf_vars.append(self.num_filter_units)
+
+        @self.Constraint(self.flowsheet().config.time,
+                         doc="Air flow rate constraint")
+        def air_flow_rate_constraint(b, t):
+            q_in = pyunits.convert(b.properties_in[t].flow_vol,
+                                   to_units=pyunits.m**3/pyunits.hour)
+            return (b.air_flow_rate[t] ==
+                    b.air_water_ratio[t] * q_in)
+
+        @self.Constraint(self.flowsheet().config.time,
+                         doc="Electricity intensity constraint")
+        def electricity_intensity_constraint(b, t):
+            q_in = pyunits.convert(b.properties_in[t].flow_vol,
+                                   to_units=pyunits.m**3/pyunits.hour)
+            blower_power = pyunits.convert(b.elec_coeff * b.air_flow_rate[t],
+                                           to_units=pyunits.kW)
+            return b.electricity_intensity[t] == blower_power/q_in
```

Comment:
I suggest reformulating this to `electricity_intensity = elec_coeff*air_water_ratio` to avoid multiplying and then dividing by `q_in`.

Reply:
I reformulated as suggested. I still keep the `air_flow_rate` variable in case of future possible use.